tests/core/eth-module/test_transactions.py

```diff
@@ -1,6 +1,10 @@
 import pytest
 
+from web3._utils.ens import (
```

Comment:
Random plumbing work that should be done at some point.  Anything under the `web3` namespace like this that is purely a testing tool should get moved to live under `web3.tools`